models/Import/Service/ResultImporter.php

```diff
@@ -356,4 +366,38 @@ private function getResultStorage(): AbstractRdsResultStorage
             )
         );
     }
+
+    /**
+     * Zero don't change SCORE_TOTAL value, but we want to update its timestamp when we have new value
+     * @param ImportResultInput $input
+     * @return bool
+     * @throws ImportResultException
+     * @throws common_exception_Error
+     */
+    private function checkOutcomeWithNewZeroValue(ImportResultInput $input): bool
```

Comment:
@Karol-Stelmaczonek I can not understand what this function checks from it's name or even from phpdoc